pipelines/build/common/build_base_file.groovy

```diff
@@ -223,15 +223,19 @@ class Builder implements Serializable {
     */
     Boolean getReproducibleCompare(Map<String, ?> configuration, String variant) {
         Boolean enableReproducibleCompare = DEFAULTS_JSON['testDetails']['enableReproducibleCompare'] as Boolean
-        if (configuration.containsKey('reproducibleCompare')) {
-            def reproducibleCompare
-            if (isMap(configuration.reproducibleCompare)) {
-                reproducibleCompare = (configuration.reproducibleCompare as Map).get(variant)
-            } else {
-                reproducibleCompare = configuration.reproducibleCompare
-            }
-            if (reproducibleCompare != null) {
-                enableReproducibleCompare = reproducibleCompare
+        if ( env.JOB_NAME.contains('pr-tester') || env.JOB_NAME.contains('release')) {
+            enableReproducibleCompare = false
+        } else {
```

Comment:
personally speaking, I feel this logic should be added when the job is generated not the build is called. e.g in [getReproducibleCompare](https://github.com/adoptium/ci-jenkins-pipelines/blob/b7e00a70ab650a485c47120deab22cbc001e9ad4/pipelines/build/common/config_regeneration.groovy#L342)

Reply:
Configurations in https://github.com/adoptium/ci-jenkins-pipelines/blob/b7e00a70ab650a485c47120deab22cbc001e9ad4/pipelines/build/common/config_regeneration.groovy are used as a placeholder it could be overwritten during the build time.